<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowTitle">
   <string>Connexion</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <spacer name="topSpacer">
     <property name="orientation"><enum>Qt::Vertical</enum></property>
     <property name="sizeHint" stdset="0"><size><width>20</width><height>40</height></size></property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="centerHorizontalLayout">
     <item>
      <spacer name="leftSpacer">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="login_card">
       <property name="minimumSize"><size><width>380</width><height>0</height></size></property>
       <property name="maximumSize"><size><width>400</width><height>16777215</height></size></property>
       <layout class="QVBoxLayout" name="formLayout">
        <property name="spacing"><number>20</number></property>
        <property name="leftMargin"><number>30</number></property>
        <property name="topMargin"><number>40</number></property>
        <property name="rightMargin"><number>30</number></property>
        <property name="bottomMargin"><number>40</number></property>
        <item>
         <widget class="QLabel" name="lbl_title"><property name="text"><string>CONNEXION</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="inputsLayout">
          <property name="spacing"><number>8</number></property>
          <item><widget class="QLabel" name="lbl_email"><property name="text"><string>Adresse Courriel</string></property></widget></item>
          <item><widget class="QLineEdit" name="input_email"><property name="placeholderText"><string>votre@email.com</string></property></widget></item>
          <item><widget class="QLabel" name="lbl_password"><property name="text"><string>Mot de passe</string></property></widget></item>
          <item><widget class="QLineEdit" name="input_password"><property name="echoMode"><enum>QLineEdit::Password</enum></property><property name="placeholderText"><string>••••••••</string></property></widget></item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_submit"><property name="text"><string>Se connecter</string></property></widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="footerLayout">
          <item><widget class="QLabel" name="lbl_footer"><property name="text"><string>Pas de compte ?</string></property></widget></item>
          <item><widget class="QPushButton" name="btn_goto_signup"><property name="text"><string>S'inscrire</string></property></widget></item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation"><enum>Qt::Vertical</enum></property>
     <property name="sizeHint" stdset="0"><size><width>20</width><height>40</height></size></property>
    </spacer>
   </item>
  </layout>
 </widget>
</ui>